aptos-move/framework/aptos-framework/sources/account.spec.move

```diff
@@ -435,6 +454,8 @@ spec aptos_framework::account {
         include !exists_at(resource_addr) ==> CreateAccountAbortsIf {addr: resource_addr};
 
         ensures signer::address_of(result_1) == resource_addr;
```

Comment:
Could you add an ensures for `result_2` as well?

Reply:
fixed